### ISSUE TITLE
Uncaught SyntaxError: Unexpected token  in JSON at position 141

### ISSUE THREAD

**Issue by dougreed** [atom/ide-typescript#117]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.28.0 x64
**Electron**: 2.0.3
**OS**: Mac OS X 10.13.5
**Thrown From**: [ide-typescript](https://github.com/atom/ide-typescript) package 0.7.6

### Stack Trace

Uncaught SyntaxError: Unexpected token 
 in JSON at position 141

```
At file:///Applications/Atom.app/Contents/Resources/app.asar/static/index.html:1

SyntaxError: Unexpected token 
 in JSON at position 141
    at JSON.parse (<anonymous>)
    at StreamMessageReader.onData (/packages/ide-typescript/node_modules/vscode-jsonrpc/lib/messageReader.js:219:29)
    at /packages/ide-typescript/node_modules/vscode-jsonrpc/lib/messageReader.js:185:19)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at Pipe.onread (net.js:594:20)
```

### Commands

```
     -6:19 find-references:activate (span.syntax--entity.syntax--name.syntax--type.syntax--class.syntax--tsx)
```

### Non-Core Packages

```
atom-ide-ui 0.12.0 
atom-typescript 12.5.3 
ide-typescript 0.7.6 
```

**Comment by rsese** [comment]
Thanks for the report! Can you confirm what were you doing at the time of the error and if the error is reproducible?

Error has popped up before but not reproducible e.g. x-ref: https://github.com/atom/ide-typescript/issues/111

**Comment by damieng** [comment]
Should be fixed with the new ide-typescript 0.8.0 update.
